packages/web3-eth-ens/package.json

```diff
@@ -13,6 +13,7 @@
     },
     "main": "src/index.js",
     "dependencies": {
+        "content-hash": "^2.5.2",
```

Comment:
I would recommend asking @ricmoo here again.. there are some cases this package wasn't doing his job nicely.. and also here.. is this external dependency required from a security perspective?

Reply:
This package is [used by @ensdomains/ui][1] for the same purpose. Intention is to make the decoding here as close to theirs as possible without pulling in lots of other deps. 

[1]: https://github.com/ensdomains/ui/blob/d2bfbe92fce7fc18e3caeb6ccec969de804b3f86/package.json#L60